GameServerLib/Items/Item.cs

```diff
@@ -39,7 +39,8 @@ public bool DecrementStackCount()
 
         public void SetStacks(byte newStacks)
         {
-            throw new System.NotImplementedException();
+            if(newStacks >= 1 && newStacks <= ItemData.MaxStack)
+                StackCount = newStacks;
```

Comment:
It's technically an integrity error if you attempt to set stack count below 1 or above item's max stack size, so I would throw an exception or at the very least raise a warning when attempting to do that.